Server/Hambasafe.Services/Services/EventLocationService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Hambasafe.DataLayer;
+using Hambasafe.DataLayer.Entities;
+using Microsoft.Data.Entity;
+
+namespace Hambasafe.Services.Services
+{
+    public interface IEventLocationService
+    {
+        Task<List<EventLocation>> FindAll();
+        Task<List<EventLocation>> FindBySuburb(string suburb);
+        Task<EventLocation> FindById(int id);
+    }
+
+    public class EventLocationService : IEventLocationService
+    {
+        private readonly IRepository<EventLocation> _repository;
+
+        public EventLocationService(IRepository<EventLocation> repository)
+        {
+            _repository = repository;
+        }
+
+        public async Task<List<EventLocation>> FindAll()
+        {
+            return await _repository.FindAll().ToListAsync();
+        }
+
+        public async Task<List<EventLocation>> FindBySuburb(string suburb)
+        {
+            return await _repository.FindAll(el => el.Suburb.Equals(suburb, StringComparison.OrdinalIgnoreCase))
+                                    .ToListAsync();
```

Comment:
Not really sure if the FindBySuburb should search on Contains, Equals or StartsWith - at the moment it searches on literal value of suburb